pkg/lint/runner.go

```diff
@@ -79,6 +79,7 @@ func NewRunner(cfg *config.Config, log logutils.Log, goenv *goutil.Env, es *lint
 			processors.NewSourceCode(lineCache, log.Child("source_code")),
 			processors.NewPathShortener(),
 			getSeverityRulesProcessor(&cfg.Severity, log, lineCache),
+			processors.NewPathPrefixer(cfg.Output.PathPrefix),
```

Comment:
Should this processor before or after NewPathShortener processor ?

Reply:
I don't believe so. The prefix is essentially a user override and user overrides should be preferred over default behavior like the shortener.

Put another way -- if the shortener comes after the prefixer and it removes the prefix or puts something in front of the prefix, it will have counteracted the user's express wishes.

Reply:
oh right, thanks